src/NzbDrone.Core/Notifications/Trakt/Resource/TraktAuthRefreshResource.cs

```diff
@@ -0,0 +1,17 @@
+using Newtonsoft.Json;
+
+namespace NzbDrone.Core.Notifications.Trakt.Resource
+{
+    public class TraktAuthRefreshResource
+    {
+        [JsonProperty(PropertyName = "access_token")]
+        public string AccessToken { get; set; }
+        [JsonProperty(PropertyName = "token_type")]
```

Comment:
Nit: Put a new line before the decorator, makes it easier to see things at a glance, looks like several files need the same treatment.